app/src/main/res/values/strings.xml

```diff
@@ -34,6 +34,8 @@
     <string name="event_details">Event details</string>
     <string name="see_maps">See maps</string>
     <string name="error_message">There was an error. Tap here to try again.</string>
+    <string name="in">In</string>
+    <string name="india">India</string>
```

Comment:
Add 157 other countries as well /s

Reply:
Sorry about that. It was already written in search fragment and when I create layout resources file for search result activity, then put it into strings.

Reply:
> Add 157 other countries as well /s

Sorry about that. It was already written in search fragment and when I create layout resources file for search result activity, then put it into strings.